Dockerfile

```diff
@@ -1,4 +1,7 @@
 FROM maven:3.8.6-openjdk-18-slim AS build
+
+ARG KEYCLOAK_SERVER_REALM
```

Comment:
Why are we passing down the realm during build time?
This will "hardcode" one specific realm on the image, making it part of the image itself. For something that will/could change depending on the environment, it is better to use an environment variable or even an external file configuration.

Reply:
Yup. Removed.